src/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,73 @@
+import "./_progress-bar.scss";
+
+import React from "react";
+import classNames from "classnames";
+
+export interface ProgressBarProps {
+  percentage: number;
+  style: {
+    trackColor?: string;
+    backgroundColor?: string;
+    completedColor?: string;
+  };
+  children: {
+    content?: React.ReactNode;
+    position?: "top" | "bottom" | "inside";
+  };
```

Comment:
I have never seen children to be an object instead of a React node and it looks weird. I don't think this is a good practice. It doesn't make sense to handle the positioning here. If children is provided it should be "inside" the track, "top" and "bottom" should be handled on the call-site. These are the use cases:

```
<ProgressBar>{"50% - appears within the ProgressBar"}</ProgressBar>

...

<div>
   <p>{"50% - some content above ProgressBar"}</p>
   <ProgressBar />
<div>

...

<div>
   <ProgressBar />
   <p>{"50% - some content below ProgressBar"}</p>
<div>
``` 

You can add examples of this to Storybook